examples/hello-world/Makefile.c64.defines

```diff
@@ -0,0 +1,2 @@
+DEFINES = WITH_CLIENT,WITH_DNS,WITH_PFS,WITH_LOGGING
+CONTIKI_WITH_IPV4 = 1
```

Comment:
I am honest ... just tired it :-)
I just assumed it to be an extention of the Makefile in the example directory.